tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -83,6 +83,8 @@ public AspNetBase(string sampleName, ITestOutputHelper outputHelper, string shut
 
             _clearMetaStruct = clearMetaStruct;
             SetEnvironmentVariable(ConfigurationKeys.AppSec.ApiSecurityEnabled, "false");
+            // without this, the developer exception page intercepts our blocking middleware and doesn't let us write the proper response
+            SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Production");
```

Comment:
I still don't understand this comment 😕 Are you saying blocking middleware doesn't work at all in non `Production` environments? Or just that it messes with our test snapshots?

Reply:
OK, cool, so it's just an "our tests/snapshots" then 👍